tool/util.dart

```diff
@@ -21,5 +21,11 @@ part "util/links.dart";
 
 String uuid;
 String fileUuid;
+String currentLinkDisplayName;
+String currentLinkBuild;
```

Comment:
These provide display names and real names for the currently building link. That way the helper methods can use them.